eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -129,14 +130,14 @@ instance NFData (Script era)
 -- | Both constructors know their original bytes
 instance SafeToHash (Script era) where
   originalBytes (TimelockScript t) = originalBytes t
-  originalBytes (PlutusScript bs) = fromShort bs
+  originalBytes (PlutusScript _ bs) = fromShort bs
 
-alwaysSucceeds, alwaysFails :: Natural -> Script era
-alwaysSucceeds n = PlutusScript (Plutus.alwaysSucceedingNAryFunction n)
-alwaysFails n = PlutusScript (Plutus.alwaysFailingNAryFunction n)
+alwaysSucceeds, alwaysFails :: Language -> Natural -> Script era
```

Comment:
Hm, I thought we moved these to `cardano-ledger-alonzo-test`...

Reply:
good point, these don't belong here.